src/domain/communication/room/room.service.mentions.from.text.spec.ts

```
@@ -190,19 +191,17 @@ describe('RoomServiceMentions', () => {
     const user = testData.user;
     const organization = testData.organization;
     const virtualContributor = testData.virtualContributor;
-    jest
-      .spyOn(userLookupService, 'getUserByNameIdOrFail')
+    // Direct assignment to avoid proxy issues with vi.spyOn
+    userLookupService.getUserByNameIdOrFail = vi
+      .fn()
       .mockResolvedValue(user);
```

ESLint

[error] Replace `⏎······.fn()⏎······` with `.fn()`

(prettier/prettier)